01week/datatypes.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+
+//Write a JavaScript program to display the current day and time.
+const year = new Date().getFullYear();
+const month = new Date().getMonth();
+const day = new Date().getDate();
+const hour = new Date().getHours();
+const minute = new Date().getMinutes();
+
+console.log(`The day of our lord: ${month}/${day}/${year}`)
+console.log(`The time of our life: ${hour}:${minute}`)
+
+//this is called template literal using backticks
+
+console.log("It's " + new Date().getMonth() + "/" + new Date().getDate() + "/" + new Date().getFullYear());
+
+//Write a JavaScript program to convert a number to a string.
+
+const stringMe = 15;
+const strung = stringMe.toString();
```

Comment:
This is a variable that stores the string '15'.  I need a function to turn ANY number into a string and return that string.